examples/next-login/src/pages/home/login.tsx

```diff
@@ -0,0 +1,145 @@
+import { Text, Link, Stack } from '@ultraviolet/ui'
+import { Icon } from '@ultraviolet/icons'
+import {
+  Form,
+  TextInputField,
+  Submit,
+  CheckboxField,
+  useForm,
+  FormErrors,
+} from '@ultraviolet/form'
+import { Theme, css, useTheme } from '@emotion/react'
+import { useState } from 'react'
+
+type FormValues = {
+  email: string
+  password: string
+  remember: boolean
+}
+
+const emailRegex = /^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,}$/i
```

Comment:
For global const / readonly we use full uppercase naming to identify them easily

```suggestion
const EMAIL_REGEX = /^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,}$/i
```